### ISSUE TITLE
Remove confusing information on Configuration page

### ISSUE THREAD

**Issue by Popolechien** [kiwix/cardshop#53]
(based on user feedback)
The current text on the Configurations page ``Create one for each individual deployment as all options are customizable only at this stage.
A running hotspot can't be renamed or content added to it (mostly).``

is somewhat arcane and not really useful. Remove it altogether

**Comment by rgaudin** [comment]
From user's feedback there's confusion in “can't be renamed” as:

* It can be renamed in the cardshop
* It can't be renamed on a running hotspot.

I think the latter is important to be understood so user customize it if they want to. But agrees current sentence is misleading.

**Comment by Popolechien** [comment]
Yes, I think this could be clarified in the user's manual, but considering there's nothing to be done about it in the cardshop it's largely superfluous.
